docs/csharp/misc/cs0077.md

```diff
@@ -39,7 +41,11 @@ class M
       o1 = new C();  
       o2 = new S();  
   
-      s = o2 as S;  
+      // Use pattern matching instead of as
+      if (o2 is S sValue)
+      {
+          s = sValue;
+      } 
       // CS0077, S is not a reference type.  
```

Comment:
This comment mentions the problem after the solution, which looks odd now. Let's bring back the removed line 42, and move that comment to that line. We need to keep in mind that "The following sample generates CS0077", it says right above the snippet.

Reply:
Hi @BartoszKlonowski, apologies but if you don't mind could you please help pushing it or pasting your idea in the comment? I think I got you but incase if I missed anything, we would have more iterations.

Reply:
No worries, it's just that this comment was valid with the previous version.
Thing is: usually the snippets in warnings and errors present the code that generates the error and only after suggest the solution.
What I propose is:
1. Bring back the removed line that generates the error
2. Put that comment to that line, so it describes the line with error, not the solution.

I also posted a suggestion addressing both these ☝️

Reply:
Thank you so much @BartoszKlonowski. I have applied your suggestion. :)